### DISCUSSION TITLE
Allowing recently added tab to sort in two ways as albums and songs

### DISCUSSION THREAD

**Discussion by Trip007** [ciderapp/Cider#79] [Feature Request]
The current recently added tab takes the whole album as a single entry so if I have a song which is a part of an album and I wanna go to the song from the recently added tab then I can't do that without having to go to the album, playing the song and then coming back to the library to see other songs of other albums similarly.
The recently added tab could add an option to sort in the order of songs as it does in itunes so we see seperate songs for easily playing instead of the whole album as a single entry.
It's something like this in itunes as in attached screenshot. The album is same but it shows all songs seperately.
![image](https://user-images.githubusercontent.com/56310029/148270569-ae2aeb9c-552c-42a6-8e80-53a0f1ca533f.png)

**Comment by Nosh118** [comment]
+1, this would be a major QoL improvement. Would also support playing through songs in a queue from most recently added which would be incredible.

**Reply by Trip007** [reply]
Yeah, It would be a great enhancement to support playing through songs like you mentionted. I added that part as well and added this discussion to the issues list.
